### PR TITLE
create bodytype json

### PR DESCRIPTION
**Pull request by ankitdas13**
- Changed default body type `urlencode` to `json` while create order

### CODE REVIEW

File: src/Order.php [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
+use Requests;
+
 class Order extends Entity
 {
     /**
      * @param $id Order id description
      */
     public function create($attributes = array())
     {
+        $attributes = json_encode($attributes);
```
Comment (ramth05) [inline src/Order.php:14]: @ankitdas13 , Is there any tests needs to be updated for this?

Comment (ankitdas13) [reply src/Order.php:14]: @ramth05 we dont have to update any where for this

Comment (ankitdas13) [reply src/Order.php:14]: @ramth05 added negative test cases for create order

File: tests/CoverageTest.php [hunk 2]
```diff
@@ -385,4 +385,31 @@ public function testUtilityCoverage(){
       $utility->testPaymentLinkVerification();
       $utility->testSubscriptionVerification();
     }
+
+    /**
+     * @covers \Razorpay\Api\Order::create
+     */
+    public function testSetHeaderJson(){
```
Comment (ramth05) [inline tests/CoverageTest.php:392]: start `{` from new line

File: tests/ExceptionTest.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        $payload = $this->payload();
+
+        $attribute = json_encode($payload);
+        try
+        {
+         $data = $this->api->order->create($attribute);
+
+         $this->assertTrue(is_array($data->toArray()));
+
+        }catch(Error $e){
```
Comment (ramth05) [inline tests/ExceptionTest.php:29]: `catch` must be start from new line

File: tests/ExceptionTest.php [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            throw new InvalidArgumentException($e);
+        }
+    }
+    
+    private function payload(){
+        $date = new \DateTime();
+        $receiptId = $date->getTimestamp();
+        return array(
```
Comment (ramth05) [inline tests/ExceptionTest.php:74]: @ankitdas13 , please use `[]` for array and do the proper formatting.

File: tests/CoverageTest.php (hunk 2 above)
Comment (ankitdas13) [reply tests/CoverageTest.php]: `{` now started from new line

File: tests/ExceptionTest.php (hunk 3 above)
Comment (ankitdas13) [reply tests/ExceptionTest.php]: `catch` now started from new line

File: tests/ExceptionTest.php (hunk 4 above)
Comment (ankitdas13) [reply tests/ExceptionTest.php]: changed `array()` method to `[]`

**Review by poojarazor — APPROVED** [review]
LGTM

**Pull request merged**